eng/Directory.Build.Common.targets

```diff
@@ -104,7 +104,6 @@
     <PackageReference Remove="System.Diagnostics.DiagnosticSource" />
     <PackageReference Remove="System.Net.Http" />
     <PackageReference Remove="System.Numerics.Vectors" />
-    <PackageReference Remove="System.Text.Encodings.Web" />
```

Comment:
Something is pulling an outdated reference to this package in the test packages. Still in the process of tracking down where, but for now leave the direct reference to avoid pipeline errors